worldedit-fabric/src/main/java/com/sk89q/worldedit/fabric/FabricPlayer.java

```diff
@@ -128,6 +130,11 @@ public void dispatchCUIEvent(CUIEvent event) {
         this.player.networkHandler.sendPacket(packet);
     }
 
+    @Override
+    public Locale getLocale() {
+        return Locale.forLanguageTag(((AccessorServerPlayerEntity) this.player).getClientLanguage().replace("_", "-"));
```

Comment:
IIRC this doesn't work, Mixin interfaces don't exist at runtime, only their implemented interfaces. So you would want to create a `ServerPlayerEntityAccess` interface, list the `String getClientLanguage()` method there (NO mixin annotations), then implement that in `MixinServerPlayerEntity` using `@Accessor`. Then here, you cast to `ServerPlayerEntityAccess` and can call the method.

Reply:
I think this is the reason I used the separate accessor mixin - as they work in this situation